Library/Formula/git-test.rb

```diff
@@ -0,0 +1,21 @@
+class GitTest < Formula
+  homepage "https://github.com/spotify/git-test"
+  url "https://github.com/spotify/git-test/archive/v1.0.1.tar.gz"
+  sha256 "1273d97644edf3690cc0310a1f6f33856f83bf736599c54150db7a7ad9e43983"
+
+  def install
+    prefix.install "test.sh"
+    bin.install "git-test"
+    man1.install "git-test.1"
+  end
+
+  test do
+    system "git", "init"
+    ln_s "#{bin}/git-test", getwd
+    cp File.join(prefix, "test.sh"), getwd
+    chmod 0755, "test.sh"
+    system "git", "add", "test.sh"
+    system "git", "commit", "-m", "initial commit"
+    system "#{bin}/git-test", "-v", "HEAD", "--verify='./test.sh'"
```

Comment:
Need to set `ENV["TERM"]` to something (e.g. `xterm`): http://bot.brew.sh/job/Homebrew%20Pull%20Requests/23669/version=yosemite/testReport/junit/brew-test-bot/yosemite/test_git_test/